std/node/_errors.ts

```diff
@@ -342,3 +742,1535 @@ export const errorMap = new Map<number, [string, string]>(
+      this.max = max;
+    }
+  }
+}
+export class ERR_HTTP2_STREAM_CANCEL extends NodeError {
+  cause?: Error;
+  constructor(error: Error) {
+    super(
+      "ERR_HTTP2_STREAM_CANCEL",
+      typeof error.message === "string"
+        ? `The pending stream has been canceled (caused by: ${error.message})`
+        : "The pending stream has been canceled",
+    );
+    if (error) {
+      this.cause = error;
+    }
+  }
+}
+
+export class ERR_INVALID_ADDRESS_FAMILY extends NodeRangeError {
+  host: string;
+  port: number;
+  constructor(addressType: string, host: string, port: number) {
+    super(
+      "ERR_INVALID_ADDRESS_FAMILY",
+      `Invalid address family: ${addressType} ${host}:${port}`,
+    );
+    this.host = host;
+    this.port = port;
+  }
+}
+
+export class ERR_INVALID_CHAR extends NodeTypeError {
+  constructor(name: string, field?: string) {
+    super(
+      "ERR_INVALID_CHAR",
+      field
+        ? `Invalid character in ${name}`
+        : `Invalid character in ${name} ["${field}"]`,
+    );
+  }
+}
+
+export class ERR_INVALID_OPT_VALUE extends NodeTypeError {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(name: string, value: any) {
+    super(
+      "ERR_INVALID_OPT_VALUE",
+      `The value "${value}" is invalid for option "${name}"`,
+    );
+  }
+}
+
+export class ERR_INVALID_RETURN_PROPERTY extends NodeTypeError {
+  constructor(input: string, name: string, prop: string, value: string) {
+    super(
+      "ERR_INVALID_RETURN_PROPERTY",
+      `Expected a valid ${input} to be returned for the "${prop}" from the "${name}" function but got ${value}.`,
+    );
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function buildReturnPropertyType(value: any) {
+  if (value && value.constructor && value.constructor.name) {
+    return `instance of ${value.constructor.name}`;
+  } else {
+    return `type ${typeof value}`;
+  }
+}
+
+export class ERR_INVALID_RETURN_PROPERTY_VALUE extends NodeTypeError {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(input: string, name: string, prop: string, value: any) {
+    super(
+      "ERR_INVALID_RETURN_PROPERTY_VALUE",
+      `Expected ${input} to be returned for the "${prop}" from the "${name}" function but got ${
+        buildReturnPropertyType(value)
+      }.`,
+    );
+  }
+}
+
+export class ERR_INVALID_RETURN_VALUE extends NodeTypeError {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(input: string, name: string, value: any) {
+    super(
+      "ERR_INVALID_RETURN_VALUE",
+      `Expected ${input} to be returned from the "${name}" function but got ${
+        buildReturnPropertyType(value)
+      }.`,
+    );
+  }
+}
+
+export class ERR_INVALID_URL extends NodeTypeError {
+  input: string;
+  constructor(input: string) {
+    super(
+      "ERR_INVALID_URL",
+      `Invalid URL: ${input}`,
+    );
+    this.input = input;
+  }
+}
```

Comment:
Are all errors from this file used in this PR? If not I suggest to open a separate PR just for errors

Reply:
Yes and yes.

I'll remove the unused errors to a different PR then